### PR TITLE
Java binding demo

### PR DESCRIPTION
**Pull request by ErisMik**

### CODE REVIEW

**Review by ksyeo1010 — APPROVED** [review]

**Comment by ksyeo1010** [comment]
just one thing , I talked with Pouya and in processFile, if it fails with invalid argument, we decided to check for extension and then tell them the file is wrong in case it got invalid argument. Not sure if this is possible with java + jni.

**Comment by ErisMik** [comment]
> just one thing , I talked with Pouya and in processFile, if it fails with invalid argument, we decided to check for extension and then tell them the file is wrong in case it got invalid argument. Not sure if this is possible with java + jni.

Should be possible at the binding level, I'll take a look at implementing.

**Comment by ErisMik** [comment]
> just one thing , I talked with Pouya and in processFile, if it fails with invalid argument, we decided to check for extension and then tell them the file is wrong in case it got invalid argument. Not sure if this is possible with java + jni.

Done

File: binding/java/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - macOS (x86_64, arm64)
+  - Windows (x86_64)
+  - Android
+  - iOS
+  - Raspberry Pi (4, 3)
+  - NVIDIA Jetson Nano
+
+## Compatibility
+
+- Java 11+
```
Comment (ksyeo1010) [inline binding/java/README.md:23]:
```suggestion
- Java 11+
- Runs on Linux (x86_64), macOS (x86_64, arm64), Windows (x86_64), Raspberry Pi (4, 3), and NVIDIA Jetson Nano.
```

**Review by ksyeo1010 — APPROVED** [review]

**Pull request merged**